cmd/bnsapi/docs/swagger.yaml

```diff
@@ -287,13 +396,46 @@ paths:
         "200":
           description: OK
           schema:
-            $ref: '#/definitions/json.RawMessage'
-        "404":
-          description: Not Found
-          schema:
-            $ref: '#/definitions/json.RawMessage'
+            $ref: '#/definitions/helpers.MultipleObjectsResponse'
+        "404": {}
         "500": {}
-      summary: Returns a list of bnsd/x/termdeposit Deposit entities.
+      summary: Returns a list of bnsd/x/termdeposit Deposit entities (individual deposits).
       tags:
       - IOV token
+  /username/owner/{address}:
+    get:
+      parameters:
+      - description: 'Address. example: C1721181E83376EF978AA4A9A38A5E27C08C7BB2 or
+          iov1c9eprq0gxdmwl9u25j568zj7ylqgc7ajyu8wxr'
+        in: path
+        name: owner
+        type: string
+      responses:
+        "200":
+          description: OK
+          schema:
+            $ref: '#/definitions/username.Token'
+        "404": {}
+        "500": {}
+      summary: Returns the username object with associated info for an owner
+      tags:
+      - Starname
+  /username/resolve/{username}:
+    get:
+      parameters:
+      - description: 'username. example: bob*iov'
```

Comment:
Please use `thematrix*iov` or other existing username that has more than one target.